app/models/user.rb

```diff
@@ -1,6 +1,29 @@
 class User < ActiveRecord::Base
+  require 'beard_trimmer'
+  require 'beard_shaver'
+  require 'beard_trimmer/with_massage'
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true, format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i }
+  validates :beard_length, numericality: { greater_than: 0 }
 
   def fullname
     first_name + " " + last_name
   end
+
+  # Update user for action that was chosen
+  def self.update_user(id, action)
```

Comment:
It is preferable that you do not tie your models directly to the actions done by the controller.  It would be preferable to have this method in the controller to use the beard trimmer do these things as business logic should not get stuffed into the model.